api/tests/test_api.py

```diff
@@ -657,18 +732,267 @@ def test_fake_eth_name_check(test_client, api_version, contract_address, token_i
+          'title': '[af498306bb191650e8614d574b3687c104bc1cd7e07c522954326752c6882770].eth'},
+         FakeEthNameCheckStatus.UNKNOWN_ETH_NAME),
+        # unknown but registered #TODO 'title': '[0xaf49...2770].eth'
+        ('0X57F1887a8bf19b14fc0df6fd9b2acc9af147ea85',
+         '47192814855232171824620094590612668126513223473283784600320596656451859494352',
+         'brantly.eth',
+         {'contractMetadata.openSea.collectionName': 'ENS: '
+                                                     'Ethereum '
+                                                     'Name '
+                                                     'Service',
+          'metadata.name': 'brantly.eth',
+          'title': 'brantly.eth'},
+         FakeEthNameCheckStatus.AUTHENTIC_ETH_NAME),  # brantly.eth uppercase hex
+        # ('0X57F1887a8bf19b14fc0df6fd9b2acc9af147ea85',
+        #  '0x37bf77d30d63cbf9ddad6b3c161522c53dcdcd8177b6177c83835c5ea69a7f8f', 
+        #  FakeEthNameCheckStatus.UNKNOWN_NFT),
+        # random ENS name
+        ('0xfe4f558a0fee0657bfa044792f5545f5a8f4ecb1',
+         '1',
+         'ENS EMOJI PUNK - NICK.ETH',
+         {'contractMetadata.name': 'ENS????',
+          'contractMetadata.openSea.collectionName': 'ENS - '
+                                                     'wMZyJHiap2',
+          'metadata.name': 'ENS EMOJI PUNK - NICK.ETH',
+          'title': 'ENS EMOJI PUNK - NICK.ETH'},
+         FakeEthNameCheckStatus.IMPERSONATED_ETH_NAME),
+        # https://rarible.com/token/0xfe4f558a0fee0657bfa044792f5545f5a8f4ecb1:1
+        ('0xd4416b13d2b3a9abae7acd5d6c2bbdbe25686401',
+         '34762977820481521209114130776556072772965907316729597364642457029530388725237',
+         'dot.this.averageman.eth',
+         {'contractMetadata.name': 'NameWrapper',
+          'contractMetadata.openSea.collectionName': 'ENS: '
+                                                     'Ethereum '
+                                                     'Name '
+                                                     'Service',
+          'metadata.name': 'dot.this.averageman.eth',
+          'title': 'dot.this.averageman.eth'},
+         FakeEthNameCheckStatus.AUTHENTIC_ETH_NAME),
+        # NameWrapper https://rarible.com/token/0xd4416b13d2b3a9abae7acd5d6c2bbdbe25686401:34762977820481521209114130776556072772965907316729597364642457029530388725237
+        # pytest.param('0x495f947276749ce646f68ac8c248420045cb7b5e',
+        #              '12254154752654038808579574986506730093213650301129873673484321000757900869633',
+        #              FakeEthNameCheckStatus.IMPERSONATED_ETH_NAME,
+        #              marks=pytest.mark.xfail(reason='also delisted on opensea')),
+        # https://rarible.com/token/0x495f947276749ce646f68ac8c248420045cb7b5e:12254154752654038808579574986506730093213650301129873673484321000757900869633
+        ('0xd4416b13d2b3a9abae7acd5d6c2bbdbe25686401',
+         '0x37bf77d30d63cbf9ddad6b3c161522c53dcdcd8177b6177c83835c5ea69a7f8f',
```

Comment:
This token_id does not match the name. Should we check this or remove token_id as argument?